pkg/httpserver/httpserver.go

```diff
@@ -28,6 +28,11 @@ type HTTPServer struct {
 	server *http.Server
 }
 
+// DebugServer is the object wrapper for OSM's HTTP server class
```

Comment:
Please consider creating a new file `pkg/httpserver/debugger.go` and moving all debug related functionality from this file into `debugger.go`.